package.json

```diff
@@ -17,36 +17,42 @@
 		"development",
 		"tool"
 	],
+	"type": "module",
 	"engines": {
 		"node": ">= 16.13.2",
 		"npm": ">= 8"
 	},
 	"scripts": {
-		"test": "npm run jsdoc-generate && npm run schema-generate",
+		"test": "npm run lint && npm run jsdoc-generate && npm run schema-generate && npm run generate-cli-doc",
+		"lint": "eslint ./",
 		"jsdoc": "npm run jsdoc-generate && open-cli site/api/index.html",
-		"jsdoc-generate": "$(node -p 'path.relative(__dirname, path.dirname(require.resolve(\"@ui5/cli\"))).startsWith(\"..\") && process.exit(1); process.exit(0)') && npm run jsdoc-generate-local || npm run jsdoc-generate-workspace",
+		"jsdoc-generate": "node --experimental-import-meta-resolve ./scripts/isWorkspace.js && npm run jsdoc-generate-workspace || npm run jsdoc-generate-local",
```

Comment:
`import.meta.resolve` works with the flag, but I'm not in favour of using it here. The API might change in future node versions. Please use require.resolve instead, which should work just fine and is the same approach that we also use in other places.

Reply:
Agree. Fixed by https://github.com/SAP/ui5-tooling/pull/650/commits/519ecd25e696d3505b3a4adb4ab2c026470e3123.